UnityProject/Assets/Scripts/PlayGroups/PlayerSync.Client.cs

```diff
@@ -61,7 +61,7 @@ public partial class PlayerSync
 
 				//experiment: not enqueueing or processing action if floating.
 				//arguably it shouldn't really be like that in the future 
-				if ( !isPseudoFloatingClient && !isFloatingClient && !blockClientMovement ) {
+				if ( (!isPseudoFloatingClient && !isFloatingClient && !blockClientMovement) || (playerScript.gameObject.GetComponent<PlayerMove>().isGhost && !blockClientMovement)) {
```

Comment:
why not replace `playerScript.gameObject.GetComponent<PlayerMove>().isGhost`
with `playerMove.isGhost`?

Reply:
My bad, did not see that the class already had it defined. I'll fix it now.